app/assets/javascripts/hyrax/i18n_helper.es6

```diff
@@ -0,0 +1,23 @@
+/**
+ * Helper class for internationalization event handling
+ */
+export default class InternationalizationHelper {
+  constructor() {
+    this.addLangClickListener();
+  }
+
+  /**
+   * Handle the event of selecting a new language from top bar, language select element.
```

Comment:
Would you also add, that this updates the `html@lang` attribute which is important for screen readers?

Reply:
Sure, just added the text.